### PR TITLE
fix: rename Studio to Lab

### LINKED ISSUE #571: rename to AI Lab
The new name of the extension is `AI Lab`.  This issue is meant to track the process.

That implies changing the name in all available resources such as OCI images, docs, code, GitHub project and more.

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

It replaces the old ai-studio with the new ai-lab naming

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

it resolves #571 

### How to test this PR?

1. it should work as before

### CODE REVIEW

**Comment by lstocchi** [comment]
We have to wait that https://github.com/redhat-et/locallm/pull/99 is merged and update the catalog

**Comment by lstocchi** [comment]
> I see a couple of references in the code, mostly naming of objects and variables, for instance `studioClient` or `StudioAPI`. Not sure that's important now but may lead to confusion later on.
> 
> User-visible changes LGTM.

Yeah i'm not updating them on this PR. I'll talk with the others about how to move for the internal changes

**Review by feloy — APPROVED** [review]
LGTM for user-facing changes

**Comment by benoitf** [comment]
looks like it's missing formatting

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Default Owners
-* @projectatomic/ai-studio-reviewers
+* @containers/podman-desktop-extension-ai-lab-reviewers
```
Comment (benoitf) [inline CODEOWNERS:2]: just a FYI, I fixed the error saying it was invalid as the team was not added to this repository

**Review by benoitf — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (lstocchi) [reply CODEOWNERS:2]: Thank you!!

**Pull request merged**